gnome-keysign.py

```diff
@@ -130,7 +137,7 @@ def __init__(self, keyid, keydata):
         self.data = keydata
 
         label = Gtk.Label()
-        label.set_markup(keydata)
+        label.set_markup(keydata.replace('<', '').replace('>', ''))
```

Comment:
But if you replace the markup tags ('<', '>') then the text will not use them anymore. Here this will show `small` in the text label.